Internal Server Error: /bookings/
Traceback (most recent call last):
  File "C:\Users\Administrator\Desktop\MVP\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Administrator\Desktop\MVP\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Administrator\Desktop\MVP\venv\Lib\site-packages\django\contrib\auth\decorators.py", line 61, in _view_wrapper
    return _redirect_to_login(request)
  File "C:\Users\Administrator\Desktop\MVP\venv\Lib\site-packages\django\contrib\auth\decorators.py", line 25, in _redirect_to_login
    resolved_login_url = resolve_url(login_url or settings.LOGIN_URL)
  File "C:\Users\Administrator\Desktop\MVP\venv\Lib\site-packages\django\shortcuts.py", line 180, in resolve_url
    return reverse(to, args=args, kwargs=kwargs)
  File "C:\Users\Administrator\Desktop\MVP\venv\Lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\Desktop\MVP\venv\Lib\site-packages\django\urls\resolvers.py", line 831, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'login_user' not found. 'login_user' is not a valid view function or pattern name.
